go/vt/mysqlctl/mycnf_flag.go

```diff
@@ -63,6 +64,7 @@ const (
 func RegisterFlags() {
 	servenv.OnParse(func(fs *pflag.FlagSet) {
 		fs.IntVar(&flagServerID, "mycnf_server_id", flagServerID, "mysql server id of the server (if specified, mycnf-file will be ignored)")
+		fs.StringVar(&flagMysqlBindAddress, "mycnf_mysql_bin_address", flagMysqlBindAddress, "address mysql binds on")
```

Comment:
All the new flags we add, we use hyphens as separators instead of underscores. So the name should be `mycnf-mysql-bin-address`